packages/storybook/.storybook/main.js

```diff
@@ -1,4 +1,14 @@
 module.exports = {
   stories: ['../src/**/*.stories.(js|ts|tsx|mdx)'],
-  addons: ['@storybook/preset-typescript', '@storybook/addon-docs'],
+  addons: [
+    '@storybook/preset-typescript',
+    {
+      name: '@storybook/addon-docs',
+      options: {
+        configureJSX: true,
+        babelOptions: {},
+        sourceLoaderOptions: null,
```

Comment:
This will give us code on the docs tab of storybook
```suggestion
        sourceLoaderOptions: {
           parser: 'typescript',
        },
```